include/SFML/Graphics/Transformable.hpp

```diff
@@ -62,6 +62,17 @@ class SFML_GRAPHICS_API Transformable
     /// See the move function to apply an offset based on the previous position instead.
     /// The default position of a transformable object is (0, 0).
     ///
+    /// **Note on offsets:**
+    /// `sf::Text` may appear offset when positioned.
+    /// This is because its local bounds are influenced by font metrics (e.g., tallest characters),
+    /// so `getGlobalBounds()` may not match the position you set.
+    ///
+    /// To remove this offset, adjust the origin manually:
+    /// \code
+    /// text.setOrigin(text.getLocalBounds().position.x, text.getLocalBounds().position.y);
+    /// text.setPosition(position);
+    /// \endcode
+    ///
```

Comment:
Thank you for tackling this! I think it doesn't need it's own "heading" of sorts, the rest of the SFML docs don't use that. Additionally, I think it's important to point out the idea behind it, that we want to align the text with the baseline at all times. Finally, I think it's not really about "removing" the offset, but just taking it into account.

```suggestion
    /// Note that `sf::Text` may appear offset when positioned.
    /// This is because its local bounds are influenced by font metrics (e.g. tallest characters)
    /// to consistently align with the text's baseline. As such the `getGlobalBounds()`
    /// position may not match the position you set.
    ///
    /// To account for this offset, the local bounds need to be considered.
    /// Either by including it in the position calculation:
    /// \code
    /// text.setPosition(position - text.getLocalBounds().position);
    /// \endcode
    /// Or by adjusting the text's origin:
    /// \code
    /// text.setOrigin(text.getLocalBounds().position);
    /// text.setPosition(position);
    /// \endcode
    ///
```